docker-compose.yml

```diff
@@ -176,6 +176,40 @@ services:
       retries: 3
       start_period: 30s
 
+  kreuzberg:
+    profiles: [kreuzberg]
+    build:
+      context: packages/ocr/kreuzberg
+      dockerfile: Dockerfile
+      args:
+        <<: *build-args-common
+        KREUZBERG_API_PORT: ${KREUZBERG_API_PORT}
+    environment:
+      <<: [*proxy-common, *common-env]
+      DATA_FOLDER: /data
+      KREUZBERG_API_PORT: ${KREUZBERG_API_PORT}
+    env_file:
+      - ./.env
+    ports:
+      - "${KREUZBERG_API_PORT}:${KREUZBERG_API_PORT}"
+    volumes:
+      - ${PWD}/${DATA_FOLDER}:/data
```

Comment:
Can you remind me how your `DATA_FOLDER` env var implementation works?

Is the intention to replace `HOST_DATA_FOLDER` and `CONTAINER_DATA_FOLDER`?

Are you going to alter this for the other OCR tools, or is that for another PR?